llarp/context.cpp

```diff
@@ -433,7 +433,8 @@ extern "C"
   void
   llarp_main_signal(struct llarp_main *ptr, int sig)
   {
-    ptr->ctx->HandleSignal(sig);
+    ptr->ctx->logic->queue_func(
+        std::bind(&llarp::Context::HandleSignal, ptr->ctx.get(), sig));
```

Comment:
this is probably clearer as:

```c++
ptr->ctx->logic->queue_func([ctx=ptr->ctx.get(),sig]() { ctx->HandleSignal(sig); });
```

Reply:
for that matter, as long as queue_func takes an r-value reference it could be even cleaner to do:
`auto func = <make the lambda>;
logic->queue_func(func);`
as 2 separate lines, but either way it's a style consideration.